apps/nextjs-website/src/api/api.ts

```diff
@@ -0,0 +1,11 @@
+import { overviews } from '@/features/products';
+
+export function getOverviewPaths(): readonly string[] {
```

Comment:
I'd pass the overviews as parameter, so you have a pure function and it is easy to test it.
```suggestion
export function getOverviewPaths(overviews: ReadonlyArray<OverviewData>): readonly string[] {
```